### ISSUE TITLE
vocms26 Jobupdater and couch

### ISSUE THREAD

**Issue by scarletnorberg** [dmwm/WMCore#7489]
This has been down twice in the last three days. I have restarted it now so it might be fine but can you please look at it to check that everything is ok.
https://its.cern.ch/jira/browse/CMSCOMPPR-364
JobUpdater
2016-12-20 22:04:47,764:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue
2016-12-20 22:11:40,524:140519395776256:INFO:JobUpdaterPoller:Priorities were synchronized, wait until the next cycle
2016-12-20 22:13:40,649:140519395776256:INFO:JobUpdaterPoller:Synchronizing priorities with ReqMgr...
2016-12-20 22:13:40,662:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue
2016-12-20 22:20:13,234:140519395776256:INFO:JobUpdaterPoller:Priorities were synchronized, wait until the next cycle
2016-12-20 22:22:13,348:140519395776256:INFO:JobUpdaterPoller:Synchronizing priorities with ReqMgr...
2016-12-20 22:22:13,360:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue
2016-12-20 22:28:44,659:140519395776256:INFO:JobUpdaterPoller:Priorities were synchronized, wait until the next cycle
2016-12-20 22:30:44,774:140519395776256:INFO:JobUpdaterPoller:Synchronizing priorities with ReqMgr...
2016-12-20 22:30:44,788:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue
2016-12-20 22:37:25,142:140519395776256:INFO:JobUpdaterPoller:Priorities were synchronized, wait until the next cycle
2016-12-20 22:39:25,177:140519395776256:INFO:JobUpdaterPoller:Synchronizing priorities with ReqMgr...
2016-12-20 22:39:25,195:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue
2016-12-20 22:46:04,016:140519395776256:INFO:JobUpdaterPoller:Priorities were synchronized, wait until the next cycle
2016-12-20 22:48:04,175:140519395776256:INFO:JobUpdaterPoller:Synchronizing priorities with ReqMgr...
2016-12-20 22:48:04,183:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue
2016-12-20 22:54:48,727:140519395776256:INFO:JobUpdaterPoller:Priorities were synchronized, wait until the next cycle
2016-12-20 22:56:48,843:140519395776256:INFO:JobUpdaterPoller:Synchronizing priorities with ReqMgr...
2016-12-20 22:56:48,852:140519395776256:INFO:JobUpdaterPoller:Found 0 workflows to update in workqueue

CouchServer
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/ 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30293e3b?revs=true&open_revs=%5B%221-e19ef464f1b80aead46b7442c80b9fe0%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30293284?revs=true&open_revs=%5B%221-023c0935a83f968bb3543022a10d4ae5%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c48b5?revs=true&open_revs=%5B%221-d488069e577436ebf1c0c99d9dcaf51d%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c46e7?revs=true&open_revs=%5B%221-2815d704afbae456d939de1de5a5a15f%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17246.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c413d?revs=true&open_revs=%5B%221-3e7a0e575f123563a58fc5997ab572c6%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c3309?revs=true&open_revs=%5B%221-b13d6c57aabe34d11f039bacea9dd032%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c2f95?revs=true&open_revs=%5B%221-d6cc59ef154368f3dfc68d47ae0481f5%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c1bae?revs=true&open_revs=%5B%221-ecfa995c8da5c8c4752270bf2caf5f02%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17240.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c25c0?revs=true&open_revs=%5B%221-5f87ca8289ba53462d3d41c53609dc7d%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17245.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c0c47?revs=true&open_revs=%5B%221-294fec7dd7a5be69ef4c9e608aa72d02%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17242.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302c0220?revs=true&open_revs=%5B%221-09ede53f48c6746a624366e522fe78f7%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17246.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bf3b6?revs=true&open_revs=%5B%221-6336fd90a3b64278a09669f4691c177c%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30292e9a?revs=true&open_revs=%5B%221-d0161cb99c025e5fbe171b058229efb5%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bf32f?revs=true&open_revs=%5B%221-208d818521e11757ec5aa15719d0552a%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bef66?revs=true&open_revs=%5B%221-d1988e0f31ee12de1971add04366334b%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bec8e?revs=true&open_revs=%5B%221-03125c9b8a18337ac6ac0e4fcac5249c%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17242.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302be9d2?revs=true&open_revs=%5B%221-5da763c82a85a015efe7ff6b0e7f44ed%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17245.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3029245d?revs=true&open_revs=%5B%221-2d452a63eb733325818e05cff58513dd%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30291cbf?revs=true&open_revs=%5B%221-4e9702b20d233890ab41607173051a85%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302919d5?revs=true&open_revs=%5B%221-35aa28c77f25a7ecadba8dd0fe34cf40%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302919c1?revs=true&open_revs=%5B%221-8997226eedfee9775bf7d94da328fa55%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30290c9c?revs=true&open_revs=%5B%221-417fd8ebdd590a8c459dedfef2e15f56%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17240.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30290c6b?revs=true&open_revs=%5B%221-9c67d3ddfa3772eb0fb969022d145697%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028e8ac?revs=true&open_revs=%5B%221-1a06559e304a5ca87d553f366752e288%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17246.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302900e9?revs=true&open_revs=%5B%221-fd42dc5d35f0e97dc5a8d920ba64a103%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028f86b?revs=true&open_revs=%5B%221-5e47c17937473bf4cde48e77e706379c%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17242.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302be355?revs=true&open_revs=%5B%221-f592616c0aec54274a3e024f63d4008f%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bdd7c?revs=true&open_revs=%5B%221-fa7e0d9b02f0bf82ddc4bebb5abb6c2e%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17245.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bd06f?revs=true&open_revs=%5B%221-2f59b81e45df127a0c4a71195dc24395%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bccad?revs=true&open_revs=%5B%221-6cf69827fed8b95f0294a0cf4d567f62%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/ 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17240.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bbf36?revs=true&open_revs=%5B%221-0c8da33ba6e0f1d3e1a29d46a99f469e%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bb477?revs=true&open_revs=%5B%221-a5567d8a231877c6cc0704df850b1fbf%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302baddd?revs=true&open_revs=%5B%221-bf4c4e366dfd2661eb55fb67ba7999fd%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028e6ca?revs=true&open_revs=%5B%221-412277175d4f13160eb05181196d7393%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17246.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302bab59?revs=true&open_revs=%5B%221-dcff17d2e055b6d5f93e89e922e07777%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17242.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b9ba1?revs=true&open_revs=%5B%221-45bab78d325eae8f72db8c27dd5fab95%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b9afc?revs=true&open_revs=%5B%221-2f66fb8125974b2097ed539fe1f3094b%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17245.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028e686?revs=true&open_revs=%5B%221-e7ce6f0a638a15b1516320f930afd55b%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028dc3a?revs=true&open_revs=%5B%221-aaf2b3c932641e218341d530d88910cf%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028d05c?revs=true&open_revs=%5B%221-b348cf747a85a01d72275a8082a25825%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17240.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028c62a?revs=true&open_revs=%5B%221-80402f23726011acc76c2814ed5bf767%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028c499?revs=true&open_revs=%5B%221-ec4c6dbc9d637a876718ffc0535742ec%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028c1eb?revs=true&open_revs=%5B%221-9e6b05084c44655a783f11a9775aa395%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028b89b?revs=true&open_revs=%5B%221-0bf282488372c2e2f3667c1d6459274f%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17246.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028aa52?revs=true&open_revs=%5B%221-b40a7936d44bf89c05bcd8a3e984601e%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17245.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028a2ed?revs=true&open_revs=%5B%221-35f30bc1412bf04bcf7d9046ca3b8f00%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b9660?revs=true&open_revs=%5B%221-501efea68ddcab9391f6e34c10533ad3%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17242.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b8c6e?revs=true&open_revs=%5B%221-4f5ef5e211e9e892a2ae368fa4040900%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b7d1f?revs=true&open_revs=%5B%221-87687af9a1226c136a20c5a576eab6c8%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b6e98?revs=true&open_revs=%5B%221-e9572d8ea6adee60473867a13ce0c252%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17240.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b5fb3?revs=true&open_revs=%5B%221-27299058bd4b8674370bda83cf43afec%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b5a5e?revs=true&open_revs=%5B%221-88522d9ccacf8ad406ea9e501049bf00%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302b59d5?revs=true&open_revs=%5B%221-4e216df4e6b8a0e7ffae10417d6a9f26%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f3028a238?revs=true&open_revs=%5B%221-8133d6b680fc2558df2e86570fbefc7e%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17246.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30289c60?revs=true&open_revs=%5B%221-447e0e9a471c9ca2bd0d712fea631bba%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30288da1?revs=true&open_revs=%5B%221-7c2008e95986052d835392106e250c1e%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17242.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30287eb1?revs=true&open_revs=%5B%221-4ff8280636d77e613a3315c7e180d470%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17243.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30286d27?revs=true&open_revs=%5B%221-c5f54a8eb7ba55d1aac1b826b69304e0%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17245.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30287086?revs=true&open_revs=%5B%221-ebe0bf3bdeb287e33efcecc0e3e5e882%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17249.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302865fe?revs=true&open_revs=%5B%221-60dae105a33aafd9f370ddf03a56f316%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17240.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30285a56?revs=true&open_revs=%5B%221-d8864606020655503015872e29d1de5f%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17241.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30284c6b?revs=true&open_revs=%5B%221-e506701ca9f2aa82612e5f4743dc4b01%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f302848b5?revs=true&open_revs=%5B%221-e9df2777b6b57a010a715db1db1da0ea%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17247.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30284179?revs=true&open_revs=%5B%221-7dffd050f574fd7af289924174d93a78%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/bcc508a7e028a63705e5551f30283b48?revs=true&open_revs=%5B%221-a33e164dc6a8a3c21d34756a95452682%22%5D&latest=true 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17244.15>] 127.0.0.1 - - GET /wmagent_summary/ 200
[Tue, 20 Dec 2016 21:33:40 GMT] [info] [<0.17248.15>] 127.0.0.1 - - GET /wmagent_summary/ 200
[Tue, 20 Dec 2016 21:34:01 GMT] [info] [<0.286.0>] Starting index update for db: wmagent_jobdump/jobs idx: _design/JobDump
[Tue, 20 Dec 2016 21:34:01 GMT] [info] [<0.17256.15>] 127.0.0.1 - - GET /wmagent_jobdump%2Ffwjrs/_design/FWJRDump/_view/outputByJobID?limit=1 200
[Tue, 20 Dec 2016 21:34:01 GMT] [info] [<0.286.0>] Index update finished for db: wmagent_jobdump/jobs idx: _design/JobDump
[Tue, 20 Dec 2016 21:34:01 GMT] [info] [<0.17280.15>] 127.0.0.1 - - GET /wmagent_jobdump%2Fjobs/_design/JobDump/_view/statusByWorkflowName?limit=1 200
[Tue, 20 Dec 2016 21:34:08 GMT] [info] [<0.17253.15>] 127.0.0.1 - - PUT /wmagent_jobdump%2Fjobs/_design/JobDump/_update/stateTransition/678811?oldstate=executing&newstate=complete&location=Agent&timestamp=1482269648 201
[Tue, 20 Dec 2016 21:34:08 GMT] [info] [<0.21605.15>] 127.0.0.1 - - POST /wmagent_jobdump%2Fjobs/_bulk_docs/ 201
[Tue, 20 Dec 2016 21:35:02 GMT] [info] [<0.286.0>] Starting index update for db: wmagent_jobdump/jobs idx: _design/JobDump
[Tue, 20 Dec 2016 21:35:02 GMT] [info] [<0.17239.15>] 127.0.0.1 - - GET /wmagent_jobdump%2Ffwjrs/_design/FWJRDump/_view/outputByJobID?limit=1 200
[Tue, 20 Dec 2016 21:35:02 GMT] [info] [<0.286.0>] Index update finished for db: wmagent_jobdump/jobs idx: _design/JobDump
[Tue, 20 Dec 2016 21:35:02 GMT] [info] [<0.17238.15>] 127.0.0.1 - - GET /wmagent_jobdump%2Fjobs/_design/JobDump/_view/statusByWorkflowName?limit=1 200
[Tue, 20 Dec 2016 21:35:09 GMT] [info] [<0.21606.15>] 127.0.0.1 - - PUT /wmagent_jobdump%2Fjobs/_design/JobDump/_update/stateTransition/678725?oldstate=executing&newstate=complete&location=Agent&timestamp=1482269709 201

**Comment by ticoann** [comment]
@scarletnorberg, Scarlet, it looks OK now but if it crashes again. Please let me know.

**Comment by ticoann** [comment]
It seems it is the same problem as #7397. I put Alan's patch on couch code so we can check the error message

**Comment by amaltaro** [comment]
@scarletnorberg did couch get more stable on that node since Seangchan patched it?

@ticoann would you mind reading that issue/thread again? where the couch error is reported and the fix suggested. Honestly, my understand is that that should *not* fix this problem, but fix the error reporting to the couch logs only. However, in practice it seems to fix the couch instabilities...
In that case, we may want to push this fix upstream to cmsdist.

**Comment by ticoann** [comment]
Hi Alan, yes we should patch this in cmsdist. After the patch this didn't happen again.
Looking at the thread, it seems this is at least temporary fix. It sounds like it doesn't prevent couch gets timeout and killed but it fixes the bug when it gets restarted automatically. (I am not sure though whether I am understanding correctly)

Anyway we should patch this since it doesn't cause any problem.
https://build.opensuse.org/package/view_file/server:database/couchdb/couchdb-1.6.1-fix-restart.patch?rev=bd081d78379f6ef4ccf2ce96b0bf5aee

**Comment by ticoann** [comment]
@vkuznet, @amaltaro, @BrunoCoimbra  Bruno, Alan, Valentin, I was trying to create the patch but source is it pointing in the spec doesn't exist.

https://github.com/cms-sw/cmsdist/blob/comp_gcc493/couchdb15.spec#L2

FYI, I can get the source code from (https://github.com/apache/couchdb/archive/1.6.1.tar.gz)

And it seems we are still using couch 1.1.1 for all cmsweb the application

https://github.com/cms-sw/cmsdist/blob/comp_gcc493/couchdb.spec#L1

https://github.com/cms-sw/cmsdist/blob/comp_gcc493/workqueue.spec

https://github.com/cms-sw/cmsdist/blob/comp_gcc493/reqmgr2.spec#L11

https://github.com/dmwm/deployment/blob/master/reqmon/deploy#L8

https://github.com/dmwm/deployment/blob/master/t0_reqmon/deploy#L8

However agent uses couch15 (I am not sure how even that works if the source url is not reachable)

https://github.com/cms-sw/cmsdist/blob/comp_gcc493/wmagent.spec#L10

I thought we upgrade to 1.6.1 (which is couchdb15.spec) also we are no longer using couchdb15.spec directly but couchdb.spec should be the same as couchdb15.spec.

Am I missing something (are we actually using 1.6.1 everywhere)?

**Comment by ticoann** [comment]
@scarletnorberg, Scarlet, it seems  vocms0130 has the same problem now. I patched couch and restarts the components.
@amaltaro, Alan,  for the record I updated 
https://twiki.cern.ch/twiki/bin/view/CMSPublic/CompOpsWorkflowTeamWmAgentRealeases

**Comment by vkuznet** [comment]
Seangchan,
your finding seem to be correct. I was working with Diego on porting to couchdb
1.5/1.6 version and my work was to validate that migration 1.1 to 1.5/1.6 will
work smoothly. For that purpose we created separate couchdb15 spec. Possibly at
that time we used different source which changed now. Regarding how clients
work I think because we still have original RPM which we didn't rebuild.

But all other spec is up to maintainers of those packages. I don't I should be
the one who should change and validate workqueue/reqmon/reqmgr2/t0_reqmon/wmagent
packages.

I would suggest that we'll create new PR for couchdb15 with new source code for
1.6.1 and build new versions of aforementioned packages. Once they'll be
validated with new couchdb we can move couchdb15.spec to couchdb.spec

Best,
Valentin.
